src/cmds/init.cmd.js

```diff
@@ -1,28 +1,32 @@
 import yargsInteractive from 'yargs-interactive'
 import initCommand from '../init';
-import { testsOptions } from './options-builder';
+import { testsOptions, credentialsOptions } from './options-builder';
 
 export const command = 'init';
 
 export const desc = 'init Perfecto and Cypress configuration files';
 
 export const builder = {
   prompt: { default: true, hidden: true, boolean: true },
-  'tests.path': {...testsOptions['tests.path']}
+  'tests.path': { ...testsOptions['tests.path']},
+  ...credentialsOptions,
+  skip: { alias: 's', type: 'boolean', default: false, describe: 'Skip interactive questions' },
 };
 
 const getInitOptions = (argv) => {
   return {
-    ...(!argv?.tests?.path) && {testsPath : { type: 'input', describe: 'Enter your tests path' }},
-    cloud: { type: 'input',describe: 'Enter your Perfecto cloud name' },
-    projectName: { type: 'input', describe: 'Enter a project name for your test root folder (only if there is no package.json in this folder)' },
```

Comment:
why you delete the question about project name ?

Reply:
Project name was used in `package.json` which we don't add anymore